elife edits after review (#21)
greenelab/nature_news_manuscript: content/05.discussion.md

## Discussion
 

Science journalism is the critical conduit between the academic and public spheres and consequently shapes the public's view of science and scientists.
However, as observed in other forms of recognition in science, biases may shift coverage away from the known demographics within science [@doi:10.1101/2020.04.14.927251, @doi:10.1016/j.cell.2022.01.004].
Ideally, scientific journalism is representative of academic papers.

> Though it would be best for news coverage to promote equitable representation, at a minimum, quotes and citations would ideally match the perceived ethnic and gender demographics of scientific academia.

Reviewer: ```suggestion
Though it would be best for news coverage to promote equitable representation, at a minimum, quotes and citations would ideally match the predicted name origin and gender demographics of scientific academia.
```

<!--Furthermore, we hope that increased representation and recognition of women in science, even beyond what is observed in authorship, can increase the amount of female first and last authors such that it better reflects the general population.
In order to draw these conclusions, we analyzed the proportion of all identified quotes that were from a speaker predicted to be male compared to the proportion of first and last authors in _Nature_ predicted to be male, which similarly is a measure of scientific participation.
Using computational methods, we performed quote association and gender prediction.
We observed a strong skew towards predicted male participation across both quotes within news articles and authorship within _Nature_ and _Springer Nature_ papers.
We also identify a gender differences between first and last authors, as previously shown [@pubmed:30070216; @doi:10.1016/j.jpedsurg.2020.05.020; @doi:10.1016/j.biopsych.2019.02.010]. -->

 To further our analysis of possible coverage disparities, we looked at differences in predicted name origins of quoted and cited authors across all the processed news articles.

> Our use of name origins is a proxy for a journalist's perceived ethnicity of a primary source as signaled by an individual's name.

Reviewer: ```suggestion
Our use of name origins is a proxy for how an individual's name might suggest ethnicity to a journalist or an individual's scientific peers.
```

> We do not intend to assign an identity to an individual but to generate a broad metric to measure possible bias for particular ethnicities during journalists' primary source gathering.

Reviewer: ```suggestion
We do not intend to assign an identity to an individual but to generate a broad metric to measure possible bias during primary source gathering.
```
